README.md

```diff
@@ -7,9 +7,9 @@
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS, Javascript
+- Bootstrap
+- Git, github
 
 ## Live Demo (if available)
```

Comment:
- [ ] Kindly deploy your project and provide a working live link in your README file. Your current live demo link is not working.

Reply:
Resolved